jerry-core/ecma/builtin-objects/ecma-builtin-number-prototype.c

```diff
@@ -253,7 +253,6 @@ ecma_builtin_number_prototype_object_to_string (ecma_number_t this_arg_number, /
   if (arguments_list_len == 0
       || ecma_number_is_nan (this_arg_number)
       || ecma_number_is_infinity (this_arg_number)
-      || ecma_number_is_zero (this_arg_number)
```

Comment:
How about `Number.prototype.toString.call (-Infinity, "hello jerry")`? This should also throw RangeError. IMO this check should not be modified, just should be placed after the radix validation part.